aws-models/amplifybackend.json

```diff
@@ -1834,7 +1832,6 @@
                 "MinimumLength": {
                     "target": "com.amazonaws.amplifybackend#__double",
                     "traits": {
-                        "smithy.api#default": 0,
                         "smithy.api#documentation": "<p>The minimum length of the password used to access the backend of your Amplify project.</p>",
                         "smithy.api#jsonName": "minimumLength",
                         "smithy.api#required": {}
```

Comment:
Is it weird that defaults got removed here?

Reply:
I can confirm this is expected. These default values were erroneously added to some booleans and numbers and these removals correct that.